project_agile_scrum/views/project_agile_team_views.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+-->
+<odoo>
+    <record id="view_project_agile_team_form" model="ir.ui.view">
+        <field name="name">project_agile_team_form</field>
+        <field name="model">project.agile.team</field>
+        <field name="inherit_id" ref="project_agile.view_project_agile_team_form"/>
+        <field name="arch" type="xml">
+            <div name="button_box">
+                <button class="oe_stat_button" type="object" name="open_active_sprint" icon="fa-tasks">
+                    <field string="Active" name="active_sprint_count" widget="statinfo" />
+                </button>
+
+                <button class="oe_stat_button" type="object" name="open_future_sprints" icon="fa-tasks">
+                        <field string="Draft" name="future_sprint_count" widget="statinfo" />
+                </button>
+
+                <button class="oe_stat_button" type="object" name="open_completed_sprints" icon="fa-tasks">
+                        <field string="Completed" name="completed_sprint_count" widget="statinfo" />
+                </button>
+            </div>
+
+            <group name="left" position="inside">
+                <field name="master_id" attrs="{'invisible': [('type', '!=', 'scrum')]}" options="{'no_create_edit': True}"/>
+                <field name="sprint_sequence" attrs="{'invisible': [('type', '!=', 'scrum')]}"/>
+            </group>
+
+            <field name="email" position="after">
+                <field name="default_sprint_length" attrs="{'invisible': [('type', '!=', 'scrum')]}"/>
+                <field name="velocity" attrs="{'invisible': [('type', '!=', 'scrum')]}"/>
+            </field>
+        </field>
+    </record>
+
+
+    <!-- Agile Team Kanban View -->
+    <record model="ir.ui.view" id="project_agile_team_kanban_view">
+        <field name="name">project.agile.team.kanban</field>
+        <field name="model">project.agile.team</field>
+        <field name="inherit_id" ref="project_agile.project_agile_team_kanban_view"/>
+        <field name="arch" type="xml">
+            <field name="image_small" position="after">
+                <field name="master_id"/>
+                <field name="type"/>
+                <field name="default_sprint_length"/>
+                <field name="velocity"/>
+            </field>
+
+
+            <li name="email" position="before">
```

Comment:
Could you format a bit in order to improve readability?

Reply:
I hope it's ok now ...